main.go

```diff
@@ -288,13 +288,10 @@ func transformCompile(args []string) ([]string, error) {
 		return nil, err
 	}
 	// log.Printf("%#v", ids)
-	var files []*ast.File
-	for _, path := range paths {
-		file, err := parser.ParseFile(fset, path, nil, parser.ParseComments)
-		if err != nil {
-			return nil, err
-		}
-		files = append(files, file)
+
+	files, err := parseFilesFromPaths(paths)
+	if err != nil {
+		return nil, err
```

Comment:
undo this change; it seems unnecessary, and a separate function for just five lines is overkill